index.d.ts

```diff
@@ -28,6 +28,8 @@ export interface FontAwesomeIconProps extends BackwardCompatibleOmit<SVGAttribut
   className?: string
   color?: string
   spin?: boolean
+  spinPulse?: boolean
```

Comment:
`beatFade` and `flip` are already present.  `spinPulse` and `spinReverse` were the only animations missing.